src/management-system-v2/components/content.tsx

```diff
@@ -18,30 +19,52 @@ type ContentProps = PropsWithChildren<{
   wrapperClass?: string;
   /** Class name for the header. */
   headerClass?: string;
+
+  siderOpened?: boolean;
 }>;
 
+//TODO: open and close hamburger menu
+
 const Content: FC<ContentProps> = ({
   children,
   title,
   compact = false,
   noHeader = false,
   wrapperClass,
   headerClass,
+  siderOpened
 }) => {
   const breakpoint = Grid.useBreakpoint();
 
   return (
     <AntLayout className={cn(styles.Main, wrapperClass)}>
       {noHeader ? null : (
         <AntLayout.Header className={cn(styles.Header, headerClass)}>
+
+          {/* Add icon into header for xs screens*/}
+          {breakpoint.xs ? <div className={styles.LogoContainer}>
+            <Link href="/processes">
+              <Image
+                src={'/proceed-icon.png'}
+                alt="PROCEED Logo"
+                className={styles.Icon}
+                width={breakpoint.xs ? 85 : 160}
+                height={breakpoint.xs ? 35 : 63}
+              />
+            </Link>
+          </div> : null }
+
           <div className={styles.Title}>{title}</div>
           {breakpoint.xs ? (
             // Hamburger menu for mobile view
+            <div>
             <Button
+              className={styles.Hamburger}
               type="text"
               style={{ marginTop: '20px', marginLeft: '15px' }}
               icon={<MenuOutlined style={{ fontSize: '170%' }} />}
-            />
+              onClick={() => !siderOpened}
```

Comment:
What does the onClick Function do?

Reply:
Removed the onClick function as it's not used anymore